drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/popover/PopoverView.java

```diff
@@ -22,10 +22,26 @@
 
     enum Position {
         LEFT,
-        RIGHT
+        RIGHT,
+        BOTTOM,
```

Comment:
As we do not use `BOTTOM` anywhere, not sure if we should add it.

Reply:
@jomarko 
Done.